src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7251,7 +7254,9 @@ private ErrorPropertySymbol CreateErrorPropertySymbol(ImmutableArray<PropertySym
                     allowUnexpandedForm: allowUnexpandedForm,
                     returnRefKind: returnRefKind,
                     returnType: returnType);
-                return new MethodGroupResolution(methodGroup, null, result, analyzedArguments, methodGroup.ResultKind, sealedDiagnostics);
+
+                // Note: the MethodGroupResolution instance is responsible for freeing its copy of analyzed arguments
```

Comment:
It is also responsible for freeing the `methodGroup`. Why mention one but not the other here?

Reply:
From our discussion, I'll leave as-is.